lib/modules/hosts/imgur.js

```diff
@@ -47,8 +47,8 @@ const imgur = {
 			}],
 		},
 	},
-	baseURL: 'http://imgur.com/',
-	cdnURL: '//i.imgur.com/',
+	baseURL: 'https://imgur.com/',
+	cdnURL: 'https://i.imgur.com/',
```

Comment:
It would be nice to provide an option to allow `http://` for uses who might get firewalled via https and don't care about mixed content warnings.